example-deployment/docker/reporting-server/reporting-server.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rmf-web/reporting-server-base
```

Comment:
There seems to be a dockerfile at `packages/reporting-server` as well. Would it be possible for the example deployment to use the "official image" as is?

In the case of dashboard, it is not possible because the dashboard requires build time configuration, so  each deployment will need a different dockerfile. For api-server, it is not possible because it depends on rmf, and we haven't had any compatibility guarantee to various rmf versions, even if we have, the release of rmf is still quite slow and we often need in-development features so each deployment will need to build their own rmf.

Reply:
I'll try to make it work

Reply:
> There seems to be a dockerfile at `packages/reporting-server` as well. Would it be possible for the example deployment to use the "official image" as is?
So I tested this, and it's working but our "official" image weights 5.14GB so our reporting server ends up having 5.9 gb

Reply:
> So I tested this, and it's working but our "official" image weights 5.14GB

Do you know whats filling up all the space? I thought reporting server does not depend on ros or rmf so it should only need python and its dependencies.

Reply:
Mmmh... sorry I think I got this msg in the wrong way and I'm confused now. By "official" image I understood: e2e [docker.pkg.github.com/open-rmf/rmf-web/e2e](docker.pkg.github.com/open-rmf/rmf-web/e2e) or even `docker.pkg.github.com/open-rmf/rmf-web/rmf:nightly`. Those weights around 5.14GB. 

Are you saying that I need to create a new official image for the `reporting-server`? and use that for the `reporting-server-migrations.dockerfile`? And use the official `ros:foxy` as our base instead of  `rmf-builder`.

 Sorry for my confusion

Reply:
Yeah, in fact we might not even need to base it on `ros:foxy`. If the reporting server does not use ros, we could probably base it on `python` or `ubuntu:20.04`, those images would be much smaller.